lawnchair/src/app/lawnchair/ui/preferences/components/SuggestionsPreference.kt

```diff
@@ -23,5 +25,14 @@ fun SuggestionsPreference() {
                 context.startActivity(intent)
             },
         )
+    } else {
+        // On some devices, the Suggestions activity could not be found or PACKAGE_USAGE_STATS is not granted.
```

Comment:
This would be confusing if the user's device doesn't have a suggestions service and this toggle appears. A better way would be to investigate why your device has a suggestion service but not the activity.

Reply:
I found this issue when I set Lawnchair as the QS provider, even if I fallback to the default QS provider, suggested apps are still showing.

Reply:
We probably merge this first, for the ones who need this. Look back on this later.